src/api/uploads/__tests__/MultiputUpload-test.fixme

```diff
@@ -15,49 +15,26 @@ const config = {
 };
 let file;
 const createSessionUrl = 'https://test.box.com/createSession';
-const destinationFolderId = '123';
+const partSize = 9;
 
-describe('api/MultiputUpload', () => {
+describe.only('api/MultiputUpload', () => {
```

Comment:
Weren't you planning on putting the new changes here in a new file?

Reply:
Yes, but these changes are for fixing the old test. If you've already converted/ing these tests, I can also fix the jest tests. I'll still write new tests in a separate file as we've discussed.

Reply:
I made these changes a long time ago, and when I apply the stash git automatically applied them here, didn't bother to extract the changes out